### PR TITLE
vmui: fix display of selected value in the selector

### PR DESCRIPTION
**Pull request by Loori-R**
Fixes the issue where the selected value is not displayed in the selector in desktop version.

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Comment by hagen1778** [comment]
@Loori-R could you please add a note to changelog?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * BUGFIX: prevent from possible panic during [background merge process](https://docs.victoriametrics.com/#storage). It may occur in rare case and was introduced at [v1.85.0](https://docs.victoriametrics.com/CHANGELOG.html#v1850) when implementing [this feature](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3337).
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): stop showing `Please enter a valid Query and execute it` error message on the first load of vmui.
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): support old format of URL params. This should make compatible copying URL between vmui of versions or using `Run in VMUI` button in datasource plugin.
+* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix the display of the selected value in the selector.
```
Comment (hagen1778) [inline docs/CHANGELOG.md:24]:
```suggestion
* BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix the display of the selected value for dropdowns on `Explore` page.
```

**Review by hagen1778 — APPROVED** [review]

**Comment by hagen1778** [comment]
Thank you!

**Pull request merged**